docs/SERVICE-MESH.md

```diff
@@ -0,0 +1,242 @@
+# Open Data Hub (ODH) Installation Guide with OpenShift Service Mesh (OSSM)
+
+This guide will walk you through the installation of Open Data Hub with OpenShift Service Mesh.
+
+## Prerequisites
+
+* OpenShift cluster
+* Command Line Interface (CLI) tools
+  * `kubectl`
+  * `operator-sdk` v1.24.1 (until operator changes are merged - to build new bundles)
+
+* Pre-installed operators
+  * Openshift Service Mesh
+  * Authorino
+  * Open Data Hub
+
+* Service Mesh Control Plane configured
+  
+### Check Installed Operators
+
+You can use the following command to verify that all required operators are installed:
+
+```sh
+kubectl get operators | awk -v RS= '/servicemesh/ && /opendatahub/ && /authorino/ {exit 0} {exit 1}' || echo "Please install all required operators."
+```
+
+#### Install Required Operators
+
+The `createSubscription` function can be used to simplify the installation of required operators:
+
+```sh
+createSubscription() {
+  local name=$1
+  local source=${2:-"redhat-operators"}
+  local channel=${3:-"stable"}
+
+  echo  "Create Subscription resource for $name"
+  eval "kubectl apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: $name
+  namespace: openshift-operators
+spec:
+  channel: $channel
+  installPlanApproval: Automatic
+  name: $name
+  source: $source
+  sourceNamespace: openshift-marketplace
+EOF"    
```

Comment:
Is the subscription created in openshift-marketplace, or should it be openshift-opeators?

Reply:
Thanks for raising this concern. This just an example for devs on how to test it, not an official user guide. Perhaps it should be emphasized in the opening section.

We should make sure we refer to the right distribution channels when providing a final end user guide @cam-garrison

Reply:
as for now:
- for upstream,  our operator is called `OpenDataHub` from `community-operators` on `fast` channel
- for downstream, the operator is called `Red Hat OpenShift Data Science` from `redhat-operators `on `alpha` channel